packages/features/portfolio/src/side-drawers/withdraw.tsx

```diff
@@ -45,7 +45,6 @@ export const Withdraw = () => {
       />
       <DepositInput
         ref={currencyInputRef}
-        isWithdraw
```

Comment:
I'm not sure you can remove this prop, it is still required inside the useDepositInput hook